### PR TITLE
Settings initial steps

### PR DESCRIPTION
**Pull request by pusztaienike**

### CODE REVIEW

File: apps/sensenet/src/components/settings/index.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+export const settingsItems = [
+  {
+    name: 'configuration',
+    displayName: 'Configuration',
+    url: resolvePathParams({ path: PATHS.configuration.appPath }),
+  },
+  {
+    name: 'stats',
+    displayName: 'Stats',
```
Comment (herflis) [inline apps/sensenet/src/components/settings/index.tsx:52]: we should localize all the menu item names of the settingsItems

File: apps/sensenet/src/components/settings/index.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        return <ContentComponent rootPath={PATHS.localization.snPath} />
+      case 'configuration':
+        return <SetupComponent />
+      case 'adminui':
+        return <PersonalSettingsEditor />
+      default:
+        return (
+          <div className={clsx(globalClasses.centered, classes.underConstructionWrapper)}>
+            <SettingsIcon className={classes.underConstructionIcon} />
+            <div>Under construction</div>
```
Comment (herflis) [inline apps/sensenet/src/components/settings/index.tsx:96]: These text should also be moved to the localization file

**Review by herflis — APPROVED** [review]

File: apps/sensenet/src/components/settings/index.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  const settingsItems = [
+    {
+      name: 'configuration',
+      displayName: localizationDrawer.titles.Configuration,
+      url: resolvePathParams({ path: PATHS.configuration.appPath }),
+    },
+    {
+      name: 'stats',
+      displayName: localizationDrawer.titles.Stats,
+      url: '/settings/stats',
```
Comment (taki9) [inline apps/sensenet/src/components/settings/index.tsx:61]: `resolvePathParams({ path: PATHS.settings.appPath, { submenu: 'stats' }}),`

correct others below too

File: apps/sensenet/src/components/settings/index.tsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+            <SettingsIcon className={classes.underConstructionIcon} />
+            <div>{localizationDrawer.underConstruction}</div>
+          </div>
+        )
+    }
+  }
+
+  return (
+    <div className={clsx(globalClasses.contentWrapper, classes.settingsWrapper)} style={{ paddingLeft: 0 }}>
+      <div className={clsx(globalClasses.contentTitle, globalClasses.centeredVertical)} style={{ display: 'grid' }}>
```
Comment (taki9) [inline apps/sensenet/src/components/settings/index.tsx:105]:
```suggestion
      <div className={clsx(globalClasses.contentTitle, globalClasses.centeredVertical)}>
```

File: apps/sensenet/src/components/drawer/PermanentDrawer.tsx [hunk 5]
```diff
@@ -127,54 +129,98 @@ export const PermanentDrawer = () => {
...
+                  </NavLink>
+                )
+              )
+            })}
+          </li>
+          <li>
+            {settingsItem && (
+              <NavLink
+                aria-label={settingsItem.url}
+                to={`${settingsItem.url}stats`}
```
Comment (taki9) [inline apps/sensenet/src/components/drawer/PermanentDrawer.tsx:188]: not this way.
`to={settingsItem.url` here
and the url should be corrected on `use-drawer-items.tsx` (commented there)

File: apps/sensenet/src/hooks/use-drawer-items.tsx [hunk 6]
```diff
@@ -164,6 +138,8 @@ export const useDrawerItems = () => {
               path: (item as CustomContentDrawerItem).settings?.appPath || '',
             },
           })
+        case 'Settings':
+          return resolvePathParams({ path: PATHS.settings.appPath })
```
Comment (taki9) [inline apps/sensenet/src/hooks/use-drawer-items.tsx:142]: You should give a value to submenu which will be the default view. In PermanentDrawer it set to stats by a hacky way (I also commented there).

**Review by taki9 — APPROVED** [review]

**Pull request merged**